src/consumers/manager.rs

```diff
@@ -150,6 +151,13 @@ impl<C> ConsumerManager<C> {
                     (Some(id), prefix) => (id, prefix),
                     (None, _) => return None,
                 };
+
+                // Don't create multitenant consumers if running in single tenant mode, and vice versa
+                if multitenant_prefix.is_some() != multitenant {
+                    trace!(%lattice_id, "Skipping consumer for lattice because multitenant doesn't match");
+                    return None;
+                }
```

Comment:
Is this expected sometimes? It feels like a caller error. Should it be a `warn` instead?

Reply:
This could happen often if a multitenant wadm and a non-multitenant wadm are using the same streams, since there will be consumers for both. Perhaps a debug, but this isn't an issue as long as you are running in the mode that you are supposed to